Library/Homebrew/language/go.rb

```diff
@@ -7,7 +7,11 @@ module Go
     # The resource names should be the import name of the package,
     # e.g. `resource "github.com/foo/bar"`
     def self.stage_deps(resources, target)
-      opoo "tried to stage empty resources array" if resources.empty?
+      if resources.empty? && ENV["HOMEBREW_DEVELOPER"].nil?
+        opoo "tried to stage empty Language::Go resources array"
+      elsif resources.empty?
+        odie "tried to stage empty Language::Go resources array"
+      end
```

Comment:
Well, I hate having to parse what is basically the same condition twice, If I could have parsed it only once and then skipped the whole block. 😛